ProjectVoyage/API Managers/SpeechToTextViewController.m

```diff
@@ -0,0 +1,126 @@
+//
```

Comment:
this file is commented out: can you leave a 1-2 sentence explanation on why you commented it out on this review comment?